src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -100,20 +103,28 @@ export const ProjectOverview: FC = () => {
     },
   });
 
+  const {
+    data: engagementListData,
+    loading: engagementListLoading,
+  } = useProjectEngagementListOverviewQuery({
+    variables: {
+      input: projectId,
+    },
+  });
+
   const engagementTypeLabel = data?.project.__typename
     ? data.project.__typename === 'TranslationProject'
       ? 'Language'
       : 'Intern'
     : null;
 
-  const populationTotal =
-    data?.project.engagements.items.reduce(
-      (total, item) =>
-        item.__typename === 'LanguageEngagement'
-          ? total + (item.language.value?.population.value ?? 0)
-          : total,
-      0
-    ) || undefined;
```

Comment:
The old reduce would have the 0 value evaluated to undefined, which will not show that on the UI at all, was that on purpose?  I thought it was a mistake so I removed `|| undefined`